bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/package.scala

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.catalog
+
+/**
+ *
+ */
+package object votable {
+
+  private[votable] implicit class StringOps(s: String) {
+
+    /**
+     * A predicate that determines whether a String is both non-empty and not
+     * "NaN".
+     */
+    def nonEmptyNonNan: Boolean =
+      s.nonEmpty && !"NaN".equals(s)
```

Comment:
Hacking around the Gemini server's propensity for adding `NaN` entries to the data.

Reply:
Neat I love the votable standard